kitsune/users/models.py

```diff
@@ -72,14 +73,14 @@ class Profile(ModelBase, SearchMixin):
         validators=[TwitterValidator],
         verbose_name=_lazy("Twitter Username"),
     )
-    facebook = models.URLField(
-        max_length=255, null=True, blank=True, verbose_name=_lazy("Facebook URL")
+    community_mozilla_org = models.CharField(
+        max_length=255, null=True, blank=True, verbose_name=_lazy("Mozilla Community Username")
```

Comment:
Do we want `null=True` here and below?

From https://docs.djangoproject.com/en/2.2/ref/models/fields/#null:
> Avoid using null on string-based fields such as CharField and TextField. If a string-based field has null=True, that means it has two possible values for “no data”: NULL, and the empty string. In most cases, it’s redundant to have two possible values for “no data;” the Django convention is to use the empty string, not NULL.